src/verify-signature.ts

```diff
@@ -0,0 +1,38 @@
+import * as jose from "jose";
+
+import { createDebug } from "./debug";
+import { getJwksUrl } from "./urls";
+
+const debug = createDebug("verify-signature");
+
+/**
+ * Verify payload signature with public key of given `domain`
+ * https://docs.saleor.io/docs/3.x/developer/extending/apps/asynchronous-webhooks#payload-signature
+ *
+ * @param domain Saleor API domain
+ * @param signature Value of the saleor-signature header
+ * @param rawBody String value of request body
+ * @returns
```

Comment:
Lets remove params if they dont provide any value. In general typescript handles it. Only if param requires some deep explainer it makes sense to add it. IMO we can stick to general comment in JSDoc format